.github/workflows/diff.yaml

```diff
@@ -220,6 +230,16 @@ jobs:
           cd build
           ctest -R memgraph__simulation --output-on-failure -j$THREADS
 
+      - name: Run single benchmark test
+        run: |
+          # Activate toolchain.
+          source /opt/toolchain-v4/activate
+
+          # Run simulation tests.
+          cd tests/mgbench
+          rm -r .cache
+          ./benchmark.py accesscontrol/small --num-workers-for-import 1 --test-system-arg  "bolt-num-workers 1"
```

Comment:
There is a bug here because the flag is called `--test-system-args` + the actual value should be `"--bolt-num-workers 1"`, leave the `--` so that we can really pass any database flag without some kind of string manipulation, ofc, it's a bit more error-prone but that's should be fine.

Reply:
I think the split file can also be passed / should be passed as `--test-system-args` flag 🤔

Reply:
This was the original approach I took, but it had other disadvantages. I will investigate that again!

Reply:
that's fine (didn't know that), so leave the `--` 😄

Reply:
-> Having "--bolt-num-workers 1 --split-file /tmp/config.txt" was making benchmark.py more confused about all the flags, that was the reason
(it is possible but will require more time)